pkg/elf/usdt_args.go

```diff
@@ -5,6 +5,6 @@
 
 package elf
 
-func parseArgs(spec *UsdtSpec) error {
+func parseArgs(_ *UsdtSpec) error {
```

Comment:
FWIW, unrelated changes are better placed in separate patches.
In any case, this was already fixed by another PR: https://github.com/cilium/tetragon/pull/4127/commits/c4bbdde7e6c1d449a457cf3664e31c099adade8b.